tests/integration-tests/tests/storage/kms_key_factory.py

```diff
@@ -151,10 +61,11 @@ def _create_kms_key(self, region):
 
         # create KMS key policy
         logging.info("Attaching key policy...")
+        partition = get_arn_partition(region)
         file_loader = FileSystemLoader(pkg_resources.resource_filename(__name__, "/../../resources"))
         env = Environment(loader=file_loader, trim_blocks=True, lstrip_blocks=True)
         key_policy = env.get_template("key_policy.json").render(
-            partition=self.partition, account_id=self.account_id, iam_role_name=self.iam_role
+            partition=partition, account_id=account_id, iam_role_name=user_iam_role
```

Comment:
minor: we can write here as `partition=get_arn_partition(region)` and remove line 64